concourse/scripts/pipelines-bosh-cloudfoundry.sh

```diff
@@ -39,35 +43,40 @@ generate_manifest_file() {
        < "${SCRIPT_DIR}/../pipelines/${pipeline_name}.yml"
 }
 
-pipeline_name="create-bosh-cloudfoundry"
-generate_vars_file > /dev/null # Check for missing vars
-bash "${SCRIPT_DIR}/deploy-pipeline.sh" \
-  "${env}" "${pipeline_name}" \
-  <(generate_manifest_file) \
-  <(generate_vars_file)
+update_pipeline() {
+  pipeline_name=$1
+  case $pipeline_name in
+    create-bosh-cloudfoundry|destroy-microbosh|destroy-cloudfoundry|failure-testing)
+      generate_vars_file > /dev/null # Check for missing vars
+      bash "${SCRIPT_DIR}/deploy-pipeline.sh" \
+	"${env}" "${pipeline_name}" \
+	<(generate_manifest_file) \
+	<(generate_vars_file)
+    ;;
+    autodelete-cloudfoundry)
+      if [ ! "${DISABLE_AUTODELETE:-}" ]; then
+	bash "${SCRIPT_DIR}/deploy-pipeline.sh" \
+		"${env}" "${pipeline_name}" \
+	  "${SCRIPT_DIR}/../pipelines/${pipeline_name}.yml" \
+	  <(generate_vars_file)
```

Comment:
The indentation is a little odd here as well. There's a mixture of tabs and spaces, which makes it a little hard to read.